### DISCUSSION TITLE
Form element selector, dropdown list.

### DISCUSSION THREAD

**Discussion by vovadpua** [TotallyInformation/node-red-contrib-uibuilder#216] [Q&A] [answered]
Good afternoon, I'm new to html and css code, I liked the zero code examples. I'm using the form fill example and can't figure out if it's possible to add a selector parameter with a combobox to the object. I really miss this, I would like to see a selector - a drop-down list of such an element in the form when filling it out.

Thank you for your work.

**Comment by TotallyInformation** [comment]
Hi, thanks for getting in touch.

I agree with you absolutely about a drop-down and a combo box and they are on my backlog of things to add. They are a bit more complex (a LOT more so for the combo box) and so didn't make it into the last release for zero-code.

You can, of course, combine zero-code output with some low-code additions to add anything that is missing from `uib-element`. I've done this several times myself now and shared some examples on the Node-RED forum as well.

So expect to see at least the select drop-down in a forthcoming release. Combo boxes are a lot harder to do while keeping them accessible so might take a bit longer. 

In the meantime, if you need an example of how to add a drop-down input to a form, let me know and I will share an example flow.

**Comment by vovadpua** [comment, marked as answer]
Good afternoon, thanks for your answer, I got out of the situation by using uib-element html. Now I can create all sorts of forms and receive data in Node-Red when they are filled out.
